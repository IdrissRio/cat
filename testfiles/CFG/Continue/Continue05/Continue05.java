public class Continue05 {
  void m() {
  outer:
    for (int i = 0; i < 10; i++) {
    inner:
      switch (i) {
      case 1:
        int c;
        continue;
      case 2:
        int d;
        continue outer;
      case 3:
        int e;
        break;
      case 4:
        int f;
        break inner;
      case 5:
        int g;
        break outer;
      default:
        int k;
      }
      int y;
    }
    int z;
  }
}