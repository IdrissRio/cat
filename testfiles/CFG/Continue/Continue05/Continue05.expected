NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [IntegerLiteral[49]]
NodeName: IntegerLiteral[49] Successors: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Successors: [VarAccess[52]]
NodeName: VarAccess[52] Successors: [IntegerLiteral[53]]
NodeName: IntegerLiteral[53] Successors: [LTExpr[51]]
NodeName: LTExpr[51] Successors: [VarAccess[62], VariableDeclarator[152]]
NodeName: VarAccess[62] Successors: [ConstCase[65]]
NodeName: ConstCase[65] Successors: [VariableDeclarator[72], ConstCase[78]]
NodeName: VariableDeclarator[72] Successors: [PostIncExpr[56]]
NodeName: PostIncExpr[56] Successors: [VarAccess[52]]
NodeName: ConstCase[78] Successors: [VariableDeclarator[85], ConstCase[91]]
NodeName: VariableDeclarator[85] Successors: [PostIncExpr[56]]
NodeName: ConstCase[91] Successors: [VariableDeclarator[98], ConstCase[104]]
NodeName: VariableDeclarator[98] Successors: [VariableDeclarator[144]]
NodeName: VariableDeclarator[144] Successors: [PostIncExpr[56]]
NodeName: ConstCase[104] Successors: [VariableDeclarator[111], ConstCase[117]]
NodeName: VariableDeclarator[111] Successors: [VariableDeclarator[144]]
NodeName: ConstCase[117] Successors: [VariableDeclarator[124], DefaultCase[130]]
NodeName: VariableDeclarator[124] Successors: [VariableDeclarator[152]]
NodeName: VariableDeclarator[152] Successors: [Exit[36]]
NodeName: Exit[36] Successors: []
NodeName: DefaultCase[130] Successors: [VariableDeclarator[136]]
NodeName: VariableDeclarator[136] Successors: [VariableDeclarator[144]]

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[36] Predecessor: [VariableDeclarator[152]]
NodeName: VariableDeclarator[152] Predecessor: [LTExpr[51], VariableDeclarator[124]]
NodeName: LTExpr[51] Predecessor: [IntegerLiteral[53]]
NodeName: IntegerLiteral[53] Predecessor: [VarAccess[52]]
NodeName: VarAccess[52] Predecessor: [VariableDeclarator[46], PostIncExpr[56]]
NodeName: VariableDeclarator[46] Predecessor: [IntegerLiteral[49]]
NodeName: IntegerLiteral[49] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: PostIncExpr[56] Predecessor: [VariableDeclarator[72], VariableDeclarator[85], VariableDeclarator[144]]
NodeName: VariableDeclarator[72] Predecessor: [ConstCase[65]]
NodeName: ConstCase[65] Predecessor: [VarAccess[62]]
NodeName: VarAccess[62] Predecessor: [LTExpr[51]]
NodeName: VariableDeclarator[85] Predecessor: [ConstCase[78]]
NodeName: ConstCase[78] Predecessor: [ConstCase[65]]
NodeName: VariableDeclarator[144] Predecessor: [VariableDeclarator[98], VariableDeclarator[111], VariableDeclarator[136]]
NodeName: VariableDeclarator[98] Predecessor: [ConstCase[91]]
NodeName: ConstCase[91] Predecessor: [ConstCase[78]]
NodeName: VariableDeclarator[111] Predecessor: [ConstCase[104]]
NodeName: ConstCase[104] Predecessor: [ConstCase[91]]
NodeName: VariableDeclarator[136] Predecessor: [DefaultCase[130]]
NodeName: DefaultCase[130] Predecessor: [ConstCase[117]]
NodeName: ConstCase[117] Predecessor: [ConstCase[104]]
NodeName: VariableDeclarator[124] Predecessor: [ConstCase[117]]
